### PR TITLE
refactor: Change to use new flattened Consul paths

### LINKED ISSUE #743: Update Consul paths to access service configuration for new flattened structure
# 🚀 Refactor

### Relevant Package [**REQUIRED**]

 This feature request is for Consul key/value paths

### Description [**REQUIRED**]
The structure of configuration in Consul is changing for 3.0 to no longer have sub sections like core, appservices and devices.

### Describe the solution you'd like
 Adjust all Consul paths that access service configuration to use to new flattened structure.

### Describe alternatives you've considered
 Have you considered any alternative solutions or workarounds?

### PR DESCRIPTION
**Pull request by lenny-intel**
closes #743

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>

# Note
This PR is dependent on the following PRs being merged:
- https://github.com/edgexfoundry/go-mod-core-contracts/pull/776
- https://github.com/edgexfoundry/go-mod-bootstrap/pull/401
- https://github.com/edgexfoundry/edgex-go/pull/4243
- https://github.com/edgexfoundry/app-functions-sdk-go/pull/1238

And the dependencies propagated to:
- Device SDK
- ASC
- Device Services.

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [ ] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

**Review by cloudxxx8 — DISMISSED** [review]
LGTM

**Review by bnevis-i — DISMISSED** [review]

**Comment by lenny-intel** [comment]
@cloudxxx8 , this PR is now ready.

**Review by bnevis-i — DISMISSED** [review]

File: TAF/config/global_variables.py [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 LOG_LEVEL = "INFO"
 
-# Version in the Consul path for all services
-CONSUL_CONFIG_VERSION = "2.0"
+# Base endpoint in the Consul path for all services
+CONSUL_CONFIG_BASE_ENDPOINT = /v1/kv/edgex/v3
```
Comment (cherrycl) [inline TAF/config/global_variables.py:6]:
```suggestion
CONSUL_CONFIG_BASE_ENDPOINT = "/v1/kv/edgex/v3"
```

Comment (cherrycl) [reply TAF/config/global_variables.py:6]: https://github.com/edgexfoundry/edgex-taf/blob/8759bd2e28b253ee0e7a88d10971ac5d5af214f2/TAF/testScenarios/integrationTest/UC_metadata_notifications/metadata_notifications.robot#L101
This file also needs to be updated.

Comment (lenny-intel) [reply TAF/config/global_variables.py]: @cherrycl , fixed

**Review by cherrycl — APPROVED** [review]
LGTM

**Pull request merged**